iOS-School-02_stack.playground/Contents.swift

```diff
@@ -0,0 +1,65 @@
+// MARK: - Stack
+/// Standard functions for working with the stack.
+class Stack {
+    private var stackElements = [Int]()
+
+    /// Adds an element the top of the stack.
+    /// - Parameter element: The number `Int` to add at the top of the stack
+    /// - Complexity: O(1)
```

Comment:
nit: no issue here, just another reminder on what we have discussed sometimes it's easier to write the doc with multiline doc comment. Just don't be shy to use it:
```suggestion
    /**
     Adds an element the top of the stack.

     - Parameter element: The number `Int` to add at the top of the stack
     - Complexity: O(1)
     */
```

Reply:
Fixed.

Reply:
Again, this is not an issue. You can stick to any of options. Just be consistent.

Reply:
Ok.